src/test/java/org/swisspush/redisques/RedisQuesTest.java

```diff
@@ -278,6 +280,7 @@ public void getQueuesFilteredInvalidPattern(TestContext context) {
                 context.assertEquals(OK, message.result().body().getString(STATUS));
                 asyncEnqueue.countDown();
             });
+            Thread.sleep(10);
```

Comment:
Why is this necessary? The test will wait until the countDown has gone from 20 to 0